gui/webdisplay/src/RWebDisplayHandle.cxx

```diff
@@ -484,7 +484,7 @@ RWebDisplayHandle::ChromeCreator::ChromeCreator(bool _edge) : BrowserCreator(tru
       fBatchExec = gEnv->GetValue((fEnvPrefix + "Batch").c_str(), "$prog --headless=new --no-sandbox --no-zygote --disable-extensions --disable-gpu --disable-audio-output $geometry $url");
       fHeadlessExec = gEnv->GetValue((fEnvPrefix + "Headless").c_str(), "$prog --headless=new --no-sandbox --no-zygote --disable-extensions --disable-gpu --disable-audio-output $geometry \'$url\' &");
    }
-   fExec = gEnv->GetValue((fEnvPrefix + "Interactive").c_str(), "$prog $geometry --new-window --app=\'$url\' &");
+   fExec = gEnv->GetValue((fEnvPrefix + "Interactive").c_str(), "$prog $geometry --new-window --app=\'$url\' >/dev/null 2>/dev/null &");
```

Comment:
why not `> /dev/null 2>&1` ?

Reply:
`&` used as special symbol when running this command. Therefore just want to avoid its usage second time

Reply:
OK, fine. And BTW, `/dev/null` is Linux specific...

Reply:
And `&` doesn't do anything on Windows

Reply:
`/dev/null` should also work on Mac - I guess.

Reply:
Yes, most probably

Reply:
`&` at the end has special handling for Windows. See:

https://github.com/root-project/root/blob/master/gui/webdisplay/src/RWebDisplayHandle.cxx#L377